satzung.md

```diff
@@ -20,15 +20,15 @@ Die Organe der Fachschaft sind:
 
 ## § 3 Vollversammlung
 
-(1) Die Vollversammlung der Fachschaft ist das oberste beschlussfassende Organ der Fachschaft. In der Vollversammlung haben alle Mitglieder der Fachschaft Sitz und Stimme.
+(1) Die Vollversammlung der Fachschaft ist das oberste beschlussfassende Organ der Fachschaft. In der Vollversammlung haben alle Mitglieder der Fachschaft Sitz und Stimme sowie ein Rede- und Antragsrecht.
 
 (2) In jedem Studienjahr soll eine ordentliche Vollversammlung stattfinden. Für die Durchführung ist der Fachschaftsrat verantwortlich.
 
 (3) Außerordentliche Vollversammlungen können durch den Fachschaftsrat oder durch mindestens 15 Mitglieder der Fachschaft einberufen werden. Verantwortlich für die Durchführung sind die einberufenden Personen.
 
-(4) Vollversammlungen müssen mindestens 14 Tage vorher per E-Mail an die Fachschaft angekündigt werden. Eine ordnungsgemäß angekündigte Vollversammlung ist beschlussfähig.
+(4) Ordentliche Vollversammlungen müssen mindestens 28 Tage vorher per E-Mail an die Fachschaft angekündigt werden. Außerordentliche Vollversammlungen müssen analog mindestens 14 Tage vorher angekündigt werden. Eine ordnungsgemäß angekündigte Vollversammlung ist beschlussfähig.
 
-(5) Vollversammlungen dürfen nur während der Vorlesungszeit einberufen werden.
+(5) Vollversammlungen dürfen nur während der Vorlesungszeit einberufen werden und müssen auf dem Gelände der Digital Engineering Fakultät stattfinden.
```

Comment:
Gelände ist ne schwierige Sache finde ich, würde "in den Räumlichkeiten der DEF" vorschlagen

Reply:
Joar, sollte ja nichts dagegen sprechen. Wir machen ja selten Versammlungen draußen

Reply:
Aber jetzt vielleicht digital? Das entfacht die ganze Diskussion nochmal neu...

Reply:
insbesondere hat die def im juristischen sinne keine räumlichkeiten, weil diese dem hpi gehören.

Reply:
Also wat nu?
Räumlichkeiten der HPI gGmbH, wobei Ordentliche VVs in begründeten Ausnahmefällen auch digital abgehalten werden können?

Reply:
Ich würde da bei der Originalfassung bleiben und das nicht weiter einschränken.

Reply:
ich stimme felix zu. ist einfacher so und das genauer zu spezifizieren ist vermutlich nicht so besonders wichtig.